build/workflow/Makefile

```diff
@@ -81,6 +83,29 @@ ${TESTS}: setup-testing
 >@${MESSAGE} end "Passed." "Failed."
 >@rm -f status_code
 
+${STANDALONE_MODULE_TEST}: ${MODULE_TESTS}
+>@test_target=$$(echo $@ | sed 's/..\/..\/${TEST_LOCATION}\/${MODULE_NAME}\///g') ; \
+    test_script=$${test_target/test-/} ; \
+    test_name=$${test_script/module_tests\/no_docker_test_/} ; \
+    test_name=$${test_name/unit_tests\/no_docker_test_/} ; \
+    extension=$$(echo $$test_name | grep -o "\.sh\|\.py$$" ) ; \
+    test_name=$${test_name/$$extension/} ; \
+    test_name=$$(echo "$$test_name" | tr '_' ' ' ) ; \
```

Comment:
These lines were meant to recover the test name from the test target name, but you have named STANDALONE_MODULE_TEST with the literal name of the test file. This creates the conflict between target names and file names that I mentioned elsewhere, and it also means that the above lines are mostly unnecessary.